src/add_remove.js

```diff
@@ -0,0 +1,61 @@
+function addNewTask() {
+  const taskArray = JSON.parse(localStorage.getItem('tasks'));
+  const description = document.getElementById('task-description').value;
+  if (description !== '') {
+    taskArray.push({
+      description,
+      completed: false,
+      index: taskArray.length + 1,
+    });
+  }
+  document.getElementById('task-description').value = '';
+  localStorage.setItem('tasks', JSON.stringify(taskArray));
+}
+
+function deleteTask(task) {
+  const taskArray = JSON.parse(localStorage.getItem('tasks'));
+  const sortedTasks = taskArray.sort((a, b) => a.index - b.index);
+  sortedTasks.splice(task.index - 1, 1);
+  sortedTasks.forEach((task, index) => {
+    task.index = index + 1;
+  });
+  localStorage.setItem('tasks', JSON.stringify(sortedTasks));
+}
+
+function editTask(e, listItem, textElement, spanItem, todoList, task, render) {
```

Comment:
- [ ]  It would be better to make this function KISS so that it may have fewer parameters.